app/src/ui/branches/branches-container.tsx

```diff
@@ -174,42 +173,37 @@ export class BranchesContainer extends React.Component<
 
   private renderPullRequests() {
     const pullRequests = this.props.pullRequests
-    if (pullRequests.length) {
+    const repo = this.props.repository
+    const name = repo.gitHubRepository
+      ? repo.gitHubRepository.fullName
+      : repo.name
+    const isOnDefaultBranch =
+      this.props.defaultBranch &&
+      this.props.currentBranch &&
+      this.props.defaultBranch.name === this.props.currentBranch.name
+
+    if (this.props.isLoadingPullRequests) {
+      return <PullRequestsLoading key="prs-loading" />
+    } else {
```

Comment:
:lipstick: the `else` here is unnecessary, and would reduce a bit of nesting

Reply:
True, but this is an either-or condition and not a guard clause to make sure invariants are met. I'm fine with removing the else, but I intentionally kept it around to communicate these two conditions are related.